### ISSUE TITLE
Reinforcement-learning Task Page

### ISSUE THREAD

**Issue by merveenoyan** [huggingface/hub-docs#357]
This is the tracker for completion of task page for `reinforcement-learning` task.

List of items to contribute:
- [ ] Task description
- [ ] Subtasks (if there is any)
- [ ] Most used dataset for the task
- [ ] Most used model for the task
- [ ] Metrics that are used to evaluate the task
- [ ] Libraries used for the task
- [ ] Use cases

**Comment by osmanalenbey** [comment]
@merveenoyan I will be happy to contribute :)

**Comment by merveenoyan** [comment]
Great!!! I'd like to briefly go through on what we want to do here 🙂 Task pages are documentation related to machine learning tasks. This is primarily aimed to educate software developers who want to simply know about the task, find a model, call a pipeline and build their first PoC. Some of the task pages are empty, some are full. The empty ones are the ones we appreciate community contributions for. 
This is an example task page for text generation task: https://huggingface.co/tasks/text-generation and this is the page for RL, which is empty: https://huggingface.co/tasks/reinforcement-learning. We want community to help us fill these pages. 
 
I explained the details in this issue: https://github.com/huggingface/hub-docs/issues/369 
Would you like to fill the whole task or parts of it? Note that it might be overwhelming to take over all of it so I suggest taking a couple of parts and see if this works for you.
Please let me know of any questions if you have any 🙂

**Comment by merveenoyan** [comment]
@osmanalenbey These are wikipedia-like pages, so you can contribute basic information about a task. Some task pages are already full, you can take a look at the page of `text-classification` for inspiration: https://huggingface.co/tasks/text-classification. The list of what you can contribute is already given above. 

>  Task description
 Subtasks (if there is any)
 Most used dataset for the task
 Most used model for the task
 Metrics that are used to evaluate the task
 Libraries used for the task
 Use cases
 Small snippet for inference that demonstrates the task

**Comment by utkarsh0308** [comment]
Hello!! I would like to contribute
can add one or two parts :)

**Comment by merveenoyan** [comment]
@utkarsh0308 yes! you can! @02shanks can you both write down what you'd like to contribute to avoid duplication of efforts 🙂

**Comment by 02shanks** [comment]
I can work on Task description and then on Most used model for the task

**Comment by utkarsh0308** [comment]
@merveenoyan  I can't find the reinforcement-learning folder in my fork

**Comment by utkarsh0308** [comment]
**Added reinforcement learning use cases #388**

**Comment by utkarsh0308** [comment]
**#388 Added Reinforcement learning use cases**

**Comment by merveenoyan** [comment]
@emiliolehoucq sure! 🙂

**Comment by emiliolehoucq** [comment]
Awesome, thanks, @merveenoyan! I added to my to-do list. I'll ping you if I run into problems.

**Comment by shreyansjainn** [comment]
Hey @merveenoyan i can see the `Inference` and the `Variants` task is still open. will it be ok if i can contribute to those?

**Comment by merveenoyan** [comment]
@shreyansjainn yep, you can :)

**Comment by shreyansjainn** [comment]
@merveenoyan great, thanks, will start working on this!!! if I can ask what is generally the timeline for these tasks? this is my first time contributing to OSS.

**Comment by merveenoyan** [comment]
@shreyansjainn feel free to take your time, we have no hurry!

**Comment by osanseviero** [comment]
Lots of cool contributions! This is just missing the snippet now
